mev_inspect/schemas/blocks.py

```diff
@@ -43,11 +43,11 @@ class TraceType(Enum):
 
 
 class Trace(CamelModel):
-    action: dict
-    block_hash: str
     block_number: int
+    block_hash: Optional[str]
+    subtraces: Optional[int]
```

Comment:
there could be some weird cases where block hash is needed (e.g. reorgs) but generally should be fine to use block number. Not sure if subtraces are optional or what the motivation here is.

Reply:
These are optional here because currently the tests do not pass these as arguments in the helpers, and since the new objects extend the definition of Trace in blocks, they become optional. This can be fixed by simply including these from now on in the tests, but I wanted to avoid changing them simultaneously